ingenialink/servo.py

```diff
@@ -495,6 +491,31 @@ def store_parameters(self, subnode: Optional[int] = None) -> None:
         finally:
             time.sleep(1.5)
 
+    def _get_drive_identification(
+        self,
+        subnode: Optional[int] = None,
+    ) -> Tuple[Optional[int], Optional[int]]:
```

Comment:
prod_code and re_number will never be None

Reply:
Ups, I'll fix it.

Reply:
[5fe5b63](https://github.com/ingeniamc/ingenialink-python/pull/422/commits/5fe5b638bceea209ea251dc8dbd52a93fbe57678)